apps/documents/templatetags/react_paragraphs.py

```diff
@@ -25,7 +27,7 @@ def react_paragraphs(context, doc, module):
     context = {
         'document': document,
```

Comment:
No, it was just due to a lack of time. I think there is plenty of room for further refactoring.